docs/reference/api-conventions.asciidoc

```diff
@@ -526,7 +528,7 @@ If one of these quantities is large we'll print it out like 10m for 10,000,000 o
 when we mean 87 though. These are the supported multipliers:
 
 [horizontal]
-``::   Single
+`s`::  Single
```

Comment:
The `s` suffix is not recognized by Elasticsearch (see also https://github.com/elastic/elasticsearch/blob/869de3f160a727fedf2eb8dc41713e9dce24ba5f/server/src/main/java/org/elasticsearch/common/unit/SizeValue.java#L170-L197) so this needs to be an empty string.